templates/common/copilot-gpt-basic/README.md

```diff
@@ -1 +1,40 @@
-For testing, will be updated soon.
\ No newline at end of file
+# Overview of the Copilot GPT template
+
+## Build a Copilot GPT from OpenAPI description document
+
+With Copilot GPT, you can build a custom version of Copilot that can be used for specific scenarios, such as for specialized knowledge, implementing specific processes, or simply to save time by reusing a set of AI prompts. For example, a grocery shopping Copilot GPT can be used to create a grocery list based on a meal plan that you send to the Copilot GPT. 
+
+## Get started with the template
+
+> **Prerequisites**
+>
+> To run this app template in your local dev machine, you will need:
+>
+> - [Node.js](https://nodejs.org/), supported versions: 16, 18
+> - A [Microsoft 365 account for development](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts).
+> - [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version 5.0.0 and higher or [Teams Toolkit CLI](https://aka.ms/teamsfx-toolkit-cli)
+> - [Copilot for Microsoft 365 license](https://learn.microsoft.com/microsoft-365-copilot/extensibility/prerequisites#prerequisites)
+
+1. First, select the Teams Toolkit icon on the left in the VS Code toolbar.
+2. In the Account section, sign in with your [Microsoft 365 account](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts) if you haven't already.
+3. Create Teams app by clicking `Provision` in "Lifecycle" section.
+4. Select `Preview in Copilot (Edge)` or `Preview in Copilot (Chrome)` from the launch configuration dropdown.
+5. Open the `Copilot` app and send a prompt to trigger your GPT.
```

Comment:
I think this step need to be updated. Looking at the recordings, user need to click "..." at the top right, and select one of their GPTs.

Reply:
Sure, let me update this in the following PR